.github/scripts/check.urls.sh

```diff
@@ -56,8 +65,8 @@ if [[ ! $REPLY =~ ^[nN]$ ]]
 then
     echo;echo "Fetching URLs from production."
     prod_urls=$(get_urls_from_prod_site_map)
-    url_count=$(echo "$prod_urls" | wc -l | cut -d' ' -f1)
-    echo;echo "Checking ${url_count} URLs, be patient.  Any non 200 URLs will be listed here:"
+    url_count=$(echo "$prod_urls" | tr -cd '\n' | wc -l | tr -d ' ')
+    echo "Found ${url_count} URLs to check, be patient. Any non 200 URLs will be listed here:"
     run_checks "$prod_urls"
     echo "Successfully checked ${url_count} URLs!"
```

Comment:
```suggestion
    echo;echo "Successfully checked ${url_count} URLs!";echo;
```

Reply:
Thank you, @mrjones-plip!! I have a conflict to resolve before merging. I will work on it tomorrow first thing.